ambassador/ambassador/ir/irretrypolicy.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any, TYPE_CHECKING
+
+from ..config import Config
+from ..utils import RichStatus
+
+from .irresource import IRResource
+
+if TYPE_CHECKING:
+    from .ir import IR
+
+
+class IRRETRYPOLICY (IRResource):
```

Comment:
nit: change this to `IRRetryPolicy`
I believe you must for gotten confused with `IRCORS` being all caps, but that's because of CORS being in caps. The rest of the classes are camel case ;)

Reply:
ah yes, i guess the ircors got me there you're right. i've changed this.